### DISCUSSION TITLE
zh-cn/manual/cli/

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#163] [Comments]
# zh-cn/manual/cli/

一个使用 Rust 编写的简单 MAA 命令行工具。 功能 运行预定义或自定义的任务，例如 maa fight，maa run <task>; 使用 maa install 和 maa update 安装和更新MaaCore及资源； 使用 maa self update 更新自身。

https://docs.maa.plus/zh-cn/manual/cli/

**Comment by 1752055152** [comment]
求助：
Installing MaaCore...
Error: 拒绝访问。 (os error 5)
